lib/matplotlib/mpl-data/matplotlibrc

```diff
@@ -607,10 +607,11 @@
 ## * CONTOUR PLOTS                                                           *
 ## ***************************************************************************
 #contour.negative_linestyle: dashed  # string or on-off ink sequence
-#contour.corner_mask:        True    # {True, False, legacy}
```

Comment:
This probably needs to be in the API change as well?

Reply:
Support for `corner_mask='legacy'` was deprecated in mpl 1.5 (doc/api/prev_api_changes/api_changes_1.5.0.rst) and the code removed in mpl 2.2 (`doc/api/prev_api_changes/api_changes_2.2.0.rst`). This evidently missed the code removal.